Content/Items/Haunted/Weapons.EchochainWhip.EchochainSystem.cs

```diff
@@ -481,6 +481,11 @@ public override bool Equals(object obj)
 
 			return false;
 		}
+
+		public override int GetHashCode()
+		{
+			throw new NotImplementedException();
```

Comment:
aaack remind me to fix this later before someone tries to make an addon and this kills them mysteriously